Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -525,6 +525,42 @@ public extension UIView {
 		anchorCenterXToSuperview()
 		anchorCenterYToSuperview()
 	}
+    
+    /// SwifterSwift: Blur the background of the view
+    ///
+    /// - Parameter effectStyle: The UIBlurEffectStyle to apply to the background
+    @available(iOS 9, *) public func blurBackground(using effectStyle: UIBlurEffectStyle) {
+        
+        backgroundColor = .clear
```

Comment:
Do we have to set the `backgroundColor` to `.clear`?

Reply:
Yes, otherwise you do not get the blurred/frosted glass affect as content scrolls behind the view.